tutorials/introductory/usage.py

```diff
@@ -164,7 +164,7 @@
 ###############################################################################
 # or (pyplot-style)
 
-x = np.linspace(0, 2, 100)
+x = np.linspace(0, 2, 100)  # This is the same data from the previous plot.
```

Comment:
Use the same comment as above. "same data" is not particularly important, and it breaks the symmetry.